manifests/o/OxfordUniversityPressELT/OxfordLearnersBookshelf/5.8.1/OxfordUniversityPressELT.OxfordLearnersBookshelf.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: OxfordUniversityPressELT.OxfordLearnersBookshelf
+PackageVersion: 5.8.1
+PackageLocale: en-US
+Publisher: Oxford University Press ELT
+PackageName: Oxford Learners Bookshelf

```

```suggestion
PackageName: Oxford Learners Bookshelf 5.8.1

```

To match the ARP table.